conf/smart-site.xml

```diff
@@ -23,4 +23,11 @@
         <description>Namenode rpcserver</description>
     </property>
 
+    <!--
+    <property>
+        <name>smart.zeppelin.web.enable</name>
+        <value>true</value>
+    </property>
+    -->
+
```

Comment:
It is set as false by default. User needs to set it to true explicitly if want to enable web.